packages/shared/components/MenuSshLogin/MenuSshLogin.tsx

```diff
@@ -156,7 +156,7 @@ const Input = styled.input(
   border: 1px solid ${theme.colors.subtle};
   border-radius: 4px;
   box-sizing: border-box;
-  color: #263238;
+  color: ${theme.colors.blueGrey[900]};
```

Comment:
I wanted to check if the change from `<React.Fragment>` to `<Flex flexDirection="column">` doesn't change anything in the web UI, but when I try to use the web UI on this branch I get "`theme.colors.blueGrey` is undefined" after clicking on Connect.

Could you make sure that this change and the `<React.Fragment>` change don't cause problems for the web UI?

Reply:
I tested that change in the web UI as well...
If only I had noticed that I am testing on `localhost:8080`, not `localhost:3080` 🙄🙄

Fixed